src/odemis/acq/test/path_test.py

```diff
@@ -1315,7 +1315,7 @@ def test_set_path_stream(self):
 
         # test guess mode for spectral with *integrated* spectrometer 2
         specs = stream.SpectrumSettingsStream("test ispec2", self.ispec2, self.ispec2.data, self.ebeam)
-        sps = stream.SEMSpectrumMDStream("test sem-ispec2", [sems, specs])
+        sps = stream.SEMSpectrumMDStream("test sem-ispec2", [sems, specs])  # TODO nothing done with that
```

Comment:
Yes, indeed. Mostly just to follow the standard pattern. I'd just leave them here.